twitter_analysis/README.md

```diff
@@ -1,24 +1,17 @@
 # TweetMind
-This project is used for grabbing tweets pertaining to various topics and performs querying to extract the tweet information and stores them in MongoDB.
+TweetMind is a project that analyzes public issues and gives a biased public overview on daily global crisis. We have started with 5 topics, Academic Workers Strike,Climate Change,Russia Ukraine War,Layoffs and Gun Violence. We have a dashboard where a user can see the analysis and certain endpoints for collecting tweets, storing them in the database, converting database-fetched tweeets to a dataframe, deduplication of tweets
```

Comment:
add information of where the topics list is and the key words. Eg, topics are contained in the file "topics.txt" and the associated keywords are contained in "topic_x_keywords.txt"